app_dart/micro_services/results_pipeline/bin/server.dart

```diff
@@ -0,0 +1,63 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:appengine/appengine.dart';
+import 'package:results_pipeline/common/auth_utils.dart';
+import 'package:results_pipeline/common/logging.dart';
+import 'package:results_pipeline/common/response.dart';
+
+/// Default handler, returns a 404
```

Comment:
You may to keep server as the integration point for one or more handlers. I'd recommend to move most of this logic to a separate file.

Reply:
I had originally written it up as a separate file. I was trying this flat layout to see if most of our microservices will be super small and would they really need the added break out. 

I'm fine with just standardizing on having the handlers broken out even if they are super small. Any thoughts?

Reply:
I think there should be a generic server they all extend from (which would handle this generic logic), and then the microservices can extend it with the RPCs they support